dts/bindings/mtd/zephyr,emu-eeprom.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019 Laczen
+# SPDX-License-Identifier: Apache-2.0
+
+description: Zephyr Emulated EEPROM device
+
+compatible: "zephyr,emu-eeprom"
+
+include: eeprom-base.yaml
+
+properties:
+    size:
+      required: true
+      description: EEPROM size (byte)
```

Comment:
IMO this shouldn't have a `description` since this is just changing the `required` field of an existing `size` property that is documented in `eeprom-base.yaml`.